scss/components/_case-overview.scss

```diff
@@ -47,14 +47,18 @@ $overview-grid-cell-width: 200px;
     width: $overview-grid-cell-width;
     z-index: 5;
 
-    .material-icons {
+    .civicase__case-overview__flow-status__icon {
       color: $brand-primary;
       cursor: pointer;
       margin-left: 8px;
       position: relative;
       top: 1px;
     }
   }
+
+  span {
```

Comment:
Can we add a specific identifier for the span? looks too generic.

Reply:
this particular rule is not needed.